query/stdlib/influxdata/influxdb/source.go

```diff
@@ -0,0 +1,158 @@
+package influxdb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/plan"
+	platform "github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/query"
+	"github.com/influxdata/influxdb/tsdb/cursors"
+)
+
```

Comment:
So everything in this file will be the common parts of the various InfluxDB read operations, is that right?

Reply:
Correct. We will need a `ReadGroupSource` and a source for meta queries which will be placed here.